CadRevealFbxProvider/BatchUtils/ScaffoldPartOptimizers/ScaffoldPartOptimizer.cs

```diff
@@ -0,0 +1,10 @@
+namespace CadRevealFbxProvider.BatchUtils.ScaffoldPartOptimizers;
+
+using CadRevealComposer.Tessellation;
+
+public abstract class ScaffoldPartOptimizer
+{
+    public abstract string GetName();
```

Comment:
`GetName` could be `Name` as a Gettter property (this is how its usually is done for readonly properties in dotnet. 

`public abstract string Name {get;}`